cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,369 @@
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Summarizes which classes in a multi-label dataset appear most often mislabeled overall.
+
+    This method works by providing any one (and only one) of the following inputs:
+
+    1. ``labels`` and ``pred_probs``, or
+    2. ``confident_joint``
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+        Refer to documentation for this argument in :py:func:`filter._find_multilabel_issues_per_class <cleanlab.filter._find_multilabel_issues_per_class>` for further details.
+
+    pred_probs : np.ndarray, optional
+      Refer to documentation for this argument in :py:func:`filter._find_multilabel_issues_per_class <cleanlab.filter._find_multilabel_issues_per_class>` for further details.
+
+
+    class_names : Iterable[str], optional
+        A list or other iterable of the string class names. The list should be in the order that
+        matches the label indices. So if class 0 is 'dog' and class 1 is 'cat', then
+        ``class_names = ['dog', 'cat']``.
+
+    confident_joint : np.ndarray, optional
+      An array of shape ``(K, 2, 2)`` representing a one-vs-rest formatted confident joint for multi-label data,
+      as returned by :py:func:`count.compute_confident_joint <cleanlab.count.compute_confident_joint>`.
+      Entry ``(c, i, j)`` in this array is the number of examples confidently counted into a ``(class c, noisy label=i, true label=j)`` bin,
+      where `i, j` are either 0 or 1 to denote whether this example belongs to class `c` or not
+      (recall examples can belong to multiple classes in multi-label classification).
+
+
+    Returns
+    -------
+    common_multilabel_issues : pd.DataFrame
+        DataFrame where each row corresponds to a Class (specified as the row-index) with columns "In Given Label", "In Suggested Label", "Num Examples", "Issue Probability".
+
+        * *In Given Label*: specifies whether the Class is True/False in the given label
+        * *Class*: If class_names is provided, the 'Class' column of the DataFrame will have the class name,
+            otherwise, the values will represent the class index.
+        * *In Suggested Label*: specifies whether the Class is  True/False in the suggested label (based on model prediction)
+        * *Num Examples*: Estimated number of examples with a label issue where this Class is True/False as specified "In Given Label" but cleanlab suggests it should be as specified In Suggested Label. I.e. the number of examples in your dataset where the Class was labeled as True but likely should have been False (or vice versa).
+        * *Issue Probability*: This is the  *Num Examples* column divided by the total number of examples in the dataset. It corresponds to the relative overall frequency of each type of label issue in your dataset.
```

Comment:
```suggestion
        * *Class*: If `class_names` is provided, the "Class" column of the DataFrame will indicate the name of the class, otherwise this column contains integers representing the class index.
        * *In Given Label*: whether the Class is originally annotated True or False in the given label.
        * *In Suggested Label*: whether the Class should be True or False in the suggested label (based on model's prediction).
        * *Num Examples*: Number of examples flagged as a label issue where this Class is True/False "In Given Label" but cleanlab estimates the annotation should actually be as specified "In Suggested Label". I.e. the number of examples in your dataset where this Class was labeled as True but likely should have been False (or vice versa).
        * *Issue Probability*: The  *Num Examples* column divided by the total number of examples in the dataset; i.e. the relative overall frequency of each type of label issue in your dataset.
```